docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3.5'
+
+services:
+  telegram-bot:
+    build:
+      dockerfile: Dockerfile
+    container_name: "app"
+    ports:
+      - "8080:8080"
+    depends_on:
+      - mongo
```

Comment:
Tip: you can add a condition to check if the database container is okay - read more about healthcheck